src/subscription_manager/gui/networkConfig.py

```diff
@@ -128,12 +128,12 @@ def write_values(self, widget=None, dummy=None):
             self.proxyEntry.set_text(proxy)
             try:
                 proxy_hostname, proxy_port = proxy.split(':')
-                self.cfg.set("server", "proxy_hostname", proxy_hostname)
-                self.cfg.set("server", "proxy_port", proxy_port)
+                self.cfg.set("server", "proxy_hostname", proxy_hostname or "")
+                self.cfg.set("server", "proxy_port", proxy_port or "")
             except ValueError:
-                # no port? just write out the hostname and assume default
-                self.cfg.set("server", "proxy_hostname", proxy)
-                self.cfg.set("server", "proxy_port", rhsm.config.DEFAULT_PROXY_PORT)
+                # no port? use the default instead of storing
+                self.cfg.set("server", "proxy_hostname", proxy or "")
+                self.cfg.set("server", "proxy_port", "")
```

Comment:
Why are we unsetting proxy_port instead of setting to a default like we used to?  Most of the things that expect port parameters expect it to be an integer so I'm worried about issues with passing `""` around instead of an int.